### PR TITLE
Add info about custom speaker and custom session form to page "How to add speakers and sessions settings, call for speakers and speakers form?" #97

### PR DESCRIPTION
**Pull request by bellaphan**

### CODE REVIEW

File: event-setup/How-to-add-speakers-and-sessions-settings-call-for-speakers-and-speakers-form.md [hunk 1]
```diff
@@ -22,6 +22,7 @@ Add the sessions types that speakers/presenters can choose from. Typically, ther
 
 ![Call for Speakers](/images/how-to-call-for-speakers3.png)
```
Comment (mariobehling) [inline event-setup/How-to-add-speakers-and-sessions-settings-call-for-speakers-and-speakers-form.md:23]: Please update name following naming convention along the lines "How to add speakers and sessions settings, call for speakers and speakers form"

File: event-setup/How-to-add-speakers-and-sessions-settings-call-for-speakers-and-speakers-form.md [hunk 2]
```diff
@@ -35,5 +36,20 @@ The last step to complete before publishing the `Call for Speakers` is to define
 ### Collect Speaker Details
 ![Call for Speakers](/images/how-to-call-for-speakers5.png)
```
Comment (mariobehling) [inline event-setup/How-to-add-speakers-and-sessions-settings-call-for-speakers-and-speakers-form.md:37]: Update name please

File: event-setup/How-to-add-speakers-and-sessions-settings-call-for-speakers-and-speakers-form.md [hunk 3]
```diff
@@ -35,5 +36,20 @@ The last step to complete before publishing the `Call for Speakers` is to define
...
 
+### Custom speaker form
+In case you want to collect more information speakers, `Eventyay` custom speaker form will let you do so by simply go to `Add custom form field`, right below the `collect speakers details`. 
+![Call for Speakers](/images/How-to-add-speakers-and-sessions-settings-call-for-speakers-and-speakers-form-custom-session-form.png)
+
+Enter the information you want to collect from speakers, you also have the options to choose between `text` and `number` (you can choose the `add` button if you want to add more).
+
 ### Collect Session Details
 ![Call for Speakers](/images/how-to-call-for-speakers6.png)
```
Comment (mariobehling) [inline event-setup/How-to-add-speakers-and-sessions-settings-call-for-speakers-and-speakers-form.md:46]: Update name please

**Review by mariobehling — CHANGES_REQUESTED** [review]

**Comment by bellaphan** [comment]
I have made the changes, I will close this and create a new PR

**Pull request closed without merging**